tests/phpunit/payment-methods/test-class-wc-stripe-upe-payment-method.php

```diff
@@ -448,7 +462,7 @@ public function test_card_payment_method_capability_is_always_enabled() {
 		$wechat_pay_method        = $this->mock_payment_methods[ WC_Stripe_Payment_Methods::WECHAT_PAY ];
 		$ach_method               = $this->mock_payment_methods[ WC_Stripe_Payment_Methods::ACH ];
 		$acss_method              = $this->mock_payment_methods[ WC_Stripe_Payment_Methods::ACSS_DEBIT ];
-
```

Comment:
Nit: Should we add this line break back in to separate variable inits and assertions?